include/cupla_driver_types.hpp

```diff
@@ -62,8 +62,13 @@ using cuplaEvent_t = void*;
 /** error category for `cuplaError` */
 struct CuplaErrorCode : public std::error_category
 {
-    virtual const char *name() const noexcept override { return "cuplaError"; }
-    virtual std::string message(int ev) const override {
+    virtual char const * name() const noexcept override { return "cuplaError"; }
+    virtual std::string message(int ev) const override
```

Comment:
you would only write `override` and skip the `virtual` here:
https://stackoverflow.com/questions/39932391/virtual-override-or-both-c